### ISSUE TITLE
Selection box for isometric maps in world should be isometric

### ISSUE THREAD

**Issue by patchuby** [mapeditor/tiled#3272]
When selecting a isometric map in the world view, the selection box is orthogonal which makes it impractical to place maps next to each other : the bounding box of one map will go over another one. It makes it tricky to select the right map. I end up spacing out the maps far from each other which significantly reduces the usefulness of the world view.

**Comment by bjorn** [comment]
Yeah, essentially the world feature wasn't really written with anything else but orthogonal maps in mind. It would be nice to improve the support for isometric worlds.
